src/js/core/directives/ui-grid-column-menu.js

```diff
@@ -275,8 +265,6 @@ function ( i18nService, uiGridConstants, gridUtil ) {
      */
     repositionMenu: function( $scope, column, positionData, $elm, $columnElement ) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Angular-UI-Grid/ui-grid/file/1998102692/issues/source?bid=2955261&fileBranchId=3118992#l266)